ateam_ui/src/src/components/FieldComponent.vue

```diff
@@ -17,8 +17,8 @@ export default {
     mounted() {
         // size is based on 140 pixels per meter times the standard max field size including boundaries (13.4 x 10.4m)
         // TODO: Make this properly resize itself
-        const width = 1876;
-        const height = 1456;
+        const width = 1876*0.66;
```

Comment:
I think these lines were just changed to make it look better on the coach computer's high resolution screen. We can keep this or remove it, either way is fine

Reply:
Removed.